internal/clients/team1/client_test.go

```diff
@@ -0,0 +1,56 @@
+package team1
+
+import (
+//	"github.com/sajari/regression"
```

Comment:
Are tests working with this commented out?

Reply:
they should be, it's just goimport is really pedantic about the order of imports